### DISCUSSION TITLE
Separate audio?

### DISCUSSION THREAD

**Discussion by bearcatjamboree** [ExistentialAudio/BlackHole#617] [Q&A]
Is there a way to create an Aggregate Device using BlackHole and have audio from desktop, but have another Aggregate Device using BlackHole for Discord?  I want to separate my audio so I can control it easier in OBS, but I can't seem to find anything online for doing that.  Thanks

**Comment by devinroth** [comment]
You can use multiple instances of BlackHole. That might work.

**Reply by bearcatjamboree** [reply]
How exactly?  I tried creating an aggregate device with BlackHole2ch and it picked up the desktop audio too.

**Comment by devinroth** [comment]
BlackHole only picks up desktop audio if you set it to the default output. Do you have it set as the default output? Perhaps I don’t understand the question.

• Devin

**Reply by bearcatjamboree** [reply]
I have to pick up my desktop audio because I use a mic, headphones, and play games.  those all involve input or output that needs captured.  But, I really want a separate channel that can be muted for discord calls.  i'm not sure how to do all of that because BlackHole is currently setup to capture everything.

**Comment by devinroth** [comment]
Discord doesn’t allow you to set the output?

• Devin

**Comment by bearcatjamboree** [comment]
It's not a problem with the output going somewhere.  It needs to be something I can use as an audio source from OBS, so I can use hot-keys to enable or disable.  I can't disable BlackHole2ch because that'll disable all audio... thus, the reason to have a separate channel for Discord.

**Comment by devinroth** [comment]
Back to what I said originally. Use two versions of BlackHole.

**Reply by bearcatjamboree** [reply]
Gotcha, but I run my install through homebrew so I'm not 100% sure how to do that without issues.  Would it be sufficient to clone the following file and restart coreaudio?

/Library/Audio/Plug-Ins/HAL/BlackHole2ch.driver

**Comment by devinroth** [comment]
No that won’t work.

But on homebrew there’s 2ch 16ch and 64ch. Pick two.

• Devin

**Reply by bearcatjamboree** [reply]
oh i see.  thanks
